helm/values.yaml

```diff
@@ -11,7 +11,7 @@ initialDelaySeconds: 600
 
 image:
   repository: id-2-osm
-  tag: 'latest'
+  # tag: 'latest'
```

Comment:
Why in comment ?

Reply:
so the tag will be fetched from the chart's `appVersion` by default which is bumped by `versionrc`
`{{- default (printf "v%s" .Chart.AppVersion) .Values.image.tag }}`